src/Sulu/Bundle/PageBundle/Admin/PageAdmin.php

```diff
@@ -409,6 +410,7 @@ public function configureViews(ViewCollection $viewCollection): void
 
     public function getSecurityContexts()
     {
+        /** @var array<string, array<string, array<string, array<string>>>> */
```

Comment:
What error is shown when we are not setting this? Normally I think phpstan should understand the code so we don't need this.

Reply:
Yeah, removed this was from an earlier attempt.